api4/webhook_test.go

```diff
@@ -33,17 +33,9 @@ func TestCreateIncomingWebhook(t *testing.T) {
 	rhook, resp := th.SystemAdminClient.CreateIncomingWebhook(hook)
 	CheckNoError(t, resp)
 
-	if rhook.ChannelId != hook.ChannelId {
-		t.Fatal("channel ids didn't match")
-	}
-
-	if rhook.UserId != th.SystemAdminUser.Id {
-		t.Fatal("user ids didn't match")
-	}
-
-	if rhook.TeamId != th.BasicTeam.Id {
-		t.Fatal("team ids didn't match")
-	}
+	require.Equal(t, rhook.ChannelId, hook.ChannelId, "channel ids didn't match")
+	require.Equal(t, rhook.UserId, th.SystemAdminUser.Id, "user ids didn't match")
+	require.Equal(t, rhook.TeamId, th.BasicTeam.Id, "team ids didn't match")
```

Comment:
I think most of these (and others in this PR) can be `assert` instead of `require`. We generally try to only use `require` when we're at a step in the tests that must succeed, such as avoiding panics on further steps or checks

Reply:
also, first write the expected value and then the one to be compared. e.g:
```go
require.Equal(t, hook.ChannelId, rhook.ChannelId, "channel ids didn't match")
```